.github/workflows/check.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     with:
       additional-env-vars: |
         _R_CHECK_CRAN_INCOMING_REMOTE_=false
-        _R_CHECK_EXAMPLE_TIMING_THRESHOLD_=10
+        _R_CHECK_EXAMPLE_TIMING_THRESHOLD_=11
```

Comment:
why is this needed?

Reply:
I increased the threshold value because check failed .https://github.com/insightsengineering/teal.reporter/actions/runs/9095129970/job/24997675784.